bundles.go

```diff
@@ -72,6 +57,21 @@ func (b Bundles) Validate() error {
 	return nil
 }
 
+func CopyBundles(bundles []Bundle) []Bundle {
```

Comment:
I decided to remove this functionality from the specific type because we want to be able to copy lists of bundles regardless their specific semantics.